### ISSUE TITLE
Create wireframe mockup of metadata upload UI

### ISSUE THREAD

**Issue by dlebauer** [PecanProject/BETYdb-YABA#1]
Need to draft what the UI for an end user who wants to upload metadata to BETYdb will look like.

This will be needed for Phase II of GSOC but it will help to have this sooner. See also https://osf.io/v7f9t/wiki/GSOC%202019%20Workplan/

This may require a few iterations.

Completion criteria - a document with images of each web page in the metadata upload workflow to be implemented.

**Comment by Chris-Schnaufer** [comment]
Make a Jupityr notebook for working with the pipeline and drone data processing.

**Comment by dlebauer** [comment]
Here is a first draft of a mockup: [metadata_upload_ui_mockup.zip](https://github.com/PecanProject/BETYdb-YABA/files/3458965/metadata_upload_ui_mockup.zip)

I am going to close this issue and set up a time to review with @KristinaRiemer and @saurabh1969 ... @Chris-Schnaufer you are also welcome to take a look.

Here are the source files for the software called pencil that I used to generate the mockup: 
[metadata_upload_UI.epgz.zip](https://github.com/PecanProject/BETYdb-YABA/files/3458967/metadata_upload_UI.epgz.zip)

**Comment by dlebauer** [comment]
Estimate for the initial wireframe would probably be closer to a 2 - there is probably more work to be done as we iterate on the design that would bring it closer to a 5.

**Comment by dlebauer** [comment]
Just realized I could export to PNG ...

Note that these are a very crude approximation of the core functionality. We can discuss design, layout, content in the review.

## Welcome
![welcome_screen](https://user-images.githubusercontent.com/464871/62333440-4ba9b880-b478-11e9-9fb3-415251a9dbe9.png)

## Upload
![upload](https://user-images.githubusercontent.com/464871/62333441-4ba9b880-b478-11e9-8081-a6fcae76d047.png)
## Upload Error Page
![error_page](https://user-images.githubusercontent.com/464871/62333443-4c424f00-b478-11e9-96ef-ea3ae3269a09.png)

##  Validation / Review Page

![validation](https://user-images.githubusercontent.com/464871/62333442-4c424f00-b478-11e9-825c-d20d67fedf68.png)

## Upload Successful

![upload_successful](https://user-images.githubusercontent.com/464871/62333444-4c424f00-b478-11e9-86f5-39d7e4c8561b.png)

**Comment by tezansahu** [comment]
Hi,
I am Tezan Sahu, a third-year undergraduate from IIT Bombay. I went through this project idea for GSoC and found it very appealing to my interest. I have a decent knowledge of Python, Docker & Kubernetes & have started learning R. I have also filled the PEcAn GSoC Application Form and would like to know your email IDs so that I could send my CV for your referral.
I would also request you to guide me in understanding the background needed for contributing to this project.
Regards.
